src/server/common/constants/file-error-messages.js

```diff
@@ -1,5 +1,9 @@
 const fileErrorMessages = {
-  virus: 'The selected file contains a virus'
+  virus: 'The selected file contains a virus',
+  empty: 'The selected file is empty',
+  tooBig: 'The selected file must be smaller than ',
+  wrongType: 'The selected file must be a ',
```

Comment:
I wonder if theres something that can be done with token replacement here? Either using `string.replace()` or passing it into `Joi` for token replacement?

Reply:
yeah this is a bit of a hack for now. did wonder about having them as functions etc

Reply:
Or just a non specific : file size too big

Reply:
the messages are from the GDS guidelines:
https://design-system.service.gov.uk/components/file-upload/